scripts/editor/get-manifest-details.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Return project details from global window object.
+ *
+ * @returns {object}
+ */
+export const getProjectDetails = () => {
+	return window?.['eightshift']?.[process.env.VERSION] ?? {};
+};
+
+/**
+ * Returns project features config details.
+ *
+ * @returns {object}
+ */
+export const getConfig = () => {
+	return getProjectDetails()?.config ?? {};
+};
+
+/**
+ * Get project features config - output css variables globally.
+ *
+ * @returns {object}
+ */
+export const getConfigOutputCssVariablesGlobally = () => {
+	return getConfig().outputCssVariablesGlobally;
+};
+
+/**
+ * Set project features config - output css variables globally.
+ *
+ * @returns {object}
+ */
+export const setConfigOutputCssVariablesGlobally = (value) => {
+	getConfig().outputCssVariablesGlobally = value;
+};
+
+/**
+ * Returns global manifest details.
+ *
+ * @returns  {object}
+ */
+export const getGlobalManifest = () => {
+	return getProjectDetails()?.globalManifest ?? {};
+};
+
+/**
+ * Returns all blocks manifest array.
+ *
+ * @returns {array}
+ */
+export const getBlocksManifest = () => {
+	return getProjectDetails()?.blocks ?? [];
+};
+
+/**
+ * Returns one block manifest details.
+ *
+ * @param {string} blockName Block name to search.
+ *
+ * @returns {object}
+ */
+export const getBlockManifest = (blockName) => {
+	return getBlocksManifest().filter((item) => item.blockName === blockName) ?? {};
```

Comment:
Does this return array or an object? 

I would say that 
`.find` would be a better fit.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

```suggestion
	return getBlocksManifest().find((item) => item.blockName === blockName) ?? {};
```

Reply:
i will check if this changes anything